app/views/telephone_appointments/_step_1.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="l-column-full">
+  <%= f.hidden_field :start_at %>
+  <%= render 'hidden_fields', f: f %>
+
+  <% if @slot_assignment_failed %>
+    <div class="error-summary t-assignment-failed">
```

Comment:
As agreed in person, I've ended up creating a separate DIV for the error handling for the times, as this block isn't always rendered.